astronomer/providers/google/cloud/extractors/bigquery_extractor.py

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: Apache-2.0.
```

Comment:
do we need this license to be added? we are currently not adding any license for providers repo files.

Reply:
Removed.